clang/lib/AST/DeclPrinter.cpp

```diff
@@ -1145,13 +1145,15 @@ void DeclPrinter::VisitLinkageSpecDecl(LinkageSpecDecl *D) {
     l = "C++";
   }
 
+  bool HasMoreThanOneDecl =
+      *D->decls_begin() && D->decls_begin()->getNextDeclInContext();
```

Comment:
```suggestion
      !D->decls().empty() && (++D->decls_begin() ) != D->decls_end()`
```
WDYT?

Reply:
Sure!

For my own education though, I'd like to know the reasoning, just to be able to make the right decision in future situations.

Reply:
It just seems a bit more readable to me.  We're checking if the length of the decls is > 1, so it seemed more clear?

Though, as this isn't performance critical code, perhaps: 
`bool HasMoreThanOneDecl = std::distance(D->decls_begin(), D->decls_end()) > 1` is much more readable.

Reply:
Got it!